README.md

```diff
@@ -399,6 +399,7 @@ In addition to es_config, the following parameters allow the customization of th
       - plugin: ingest-geoip 
   ```
 * ```es_path_repo``` Sets the whitelist for allowing local back-up repositories
+* ```es_processors``` (e.g. `16`) Sets the number of processors (cores). Used in sizing thread pools. Useful for multitenancy or when the value is not correctly detected.
```

Comment:
We are planning to drop multi-tenancy support as part of the 7.x release. Given the warnings in https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-threadpool.html#processors about this being an expert level use-case I'm a bit hesitant to include a specific setting just for this. Especially when it can already be set directly in `es_config`